woocommerce/woocommerce
# Updated link href attribute in new product editor on "How to prepare images" section

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




Issue was introduced by issue #39154 

Closes #39154 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to admin dashboard
2. Make sure the [new product editor](https://woocommerce.com/document/new-product-editor-beta/) is enabled 
3. Go to product editor, navigate the image section and click on the "How to prepare images?" link



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
